cmd/kubeadm/app/cmd/token.go

```diff
@@ -381,3 +392,32 @@ func getClientset(file string, dryRun bool) (clientset.Interface, error) {
 	}
 	return kubeconfigutil.ClientSetFromFile(file)
 }
+
+func findExistingKubeConfig(file string) string {
```

Comment:
Most of this code exists in clientcmd.NewNonInteractiveDeferredLoadingClientConfig and should not be leveraged using the client tooling.